pkg/network/usm/shared_libraries_test.go

```diff
@@ -203,6 +204,15 @@ func TestSameInodeRegression(t *testing.T) {
 	}, time.Second*10, time.Second, "")
 }
 
+func syncMapLength(m *sync.Map) int {
```

Comment:
Can we add other tests to make sure that deadlocks and race conditions will not happen?

Reply:
How can you test a deadlock? or a race condition?

Reply:
We can consider performing a stress test, running multiple processes at once and see if the code handles it as expected. But feel free to resolve it, that's non blocking IMO